<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyValueWidget</class>
 <widget class="QWidget" name="KeyValueWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>36</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QCheckBox" name="chk_item_enabled">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txt_item_name">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txt_item_value"/>
   </item>
   <item>
    <widget class="QToolButton" name="btn_remove_item">
     <property name="text">
      <string>-</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/delete-48.png</normaloff>:/images/delete-48.png
      </iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
